packages/openapi-generator/src/generator.ts

```diff
@@ -239,3 +253,34 @@ export async function getSdkVersion(): Promise<string> {
     await readFile(resolve(__dirname, '../package.json'), 'utf8')
   ).version;
 }
+
+function generateAdditionalFiles(
+  additionalFiles: string,
+  serviceDir: string
+): Promise<void[]> {
+  logger.info(
```

Comment:
Isn't this more debug than info?

Reply:
I don't have strong opinion and I'm fine with both. This is just an alignment with the odata generator.